### DISCUSSION TITLE
Set Default discoutn as Individual

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#14543] [General] [answered]
Hi guys, does anyone know if it is possible to set the individual discounts as default.. at the moment the default is group discount..

![image](https://user-images.githubusercontent.com/54880467/114381660-fd374180-9ba8-11eb-93e4-65a74181bacf.png)

**Comment by rskrzypczak** [comment, marked as answer]
Currently, this is not possible.
There is no option to set a default value for this type of item.

**Reply by rioshaz** [reply]
@rskrzypczak can we do it from code side ? if u know ?
